src/app/views/auth-views/login/login.component.html

```diff
@@ -1,10 +1,34 @@
-<form>
-    <div class="form-group">
-        <label for="formGroupExampleInput">Example label</label>
-        <input type="text" class="form-control" id="formGroupExampleInput" placeholder="Example input">
+<div class="d-flex flex-wrap  justify-content-center align-items-center">
+    <div class="card m-t-50" style="width: 22rem;">
+        <div class="card-body">
+            <div class="d-flex flex-wrap  justify-content-center align-items-center">
+                <p style="font-size: medium;">Login</p>
+            </div>
+            <form>
+                <div class="form-group">
+                    <label for="exampleInputEmail1">Email </label>
+                    <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp">
+                </div>
+                <div class="form-group">
+                    <label for="exampleInputPassword1">Password</label>
+                    <input type="password" class="form-control" id="exampleInputPassword1">
+                </div>
+                <div class="d-flex justify-content-between">
+                    <div class="d-flex justify-content-start">
+                        <a [routerLink]="['../registration']" class="btn btn-link ">Registration</a>
+                    </div>
+                    <div class="d-flex justify-content-end">
+                        <a [routerLink]="['../forgot-password']" class="btn btn-link">Forgot Password?</a>
```

Comment:
Will not work please change router link value

Reply:
Reslioved.